docs/reference/errors-and-warnings/NU1204.md

```diff
@@ -0,0 +1,21 @@
+---
+title: NuGet Error NU1204
+description: NU1204 Error code
+author: nkolev92
+ms.date: 02/02/2022
+ms.topic: reference
+f1_keywords: 
+  - NU1204
+---
+
+# NuGet Warning NU1204
+
+<pre>Invalid tools package Contoso.Tool 1.0.0. Tools packages cannot contain more than one PackageType.</pre>
+
+### Issue
+
+The Contoso.Tool package has more than package type. [DotnetTool](../../create-packages/set-package-type.md#known-package-types packages can only contain one PackageType.
```

Comment:
This link seems to be broken. See the preview: https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1204?branch=pr-en-us-2659

Reply:
It does, but the review is broken. 
It doesn't really create a review link for the dotnet/core part of the docs. 

If you change that to remove the preview part of the url, you get: https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools-how-to-create which works. 

The build validates these links I think.

Reply:
Check the preview, the link doesn't have the right syntax, it's missing the training `)`, but since this comment started on an outdated commit, I can't use github's suggestion feature.

Reply:
Oh nice catch. 

Weird that the build did not validate that :(